CONTRIBUTING.md

```diff
@@ -1,28 +1,121 @@
-# How to Contribute
+# How to Contribute to the Google Ads API Client Library for Java
 
-We'd love to accept your patches and contributions to this project. There are
-just a few small guidelines you need to follow.
+## Overview: We Welcome Your Contributions
 
-## Contributor License Agreement
+Welcome and thank you for your interest in contributing to the Google Ads API Client Library for
+Java. Open source contributions are important to foster a vibrant developer community and help
+us provide a world-class developer experience. We value your contributions, no matter how small.
+Even small documentation fixes can go a long way in helping other developers use our API. Before
+contributing, please review the guidelines and best practices below.
 
-Contributions to this project must be accompanied by a Contributor License
-Agreement. You (or your employer) retain the copyright to your contribution;
-this simply gives us permission to use and redistribute your contributions as
-part of the project. Head over to <https://cla.developers.google.com/> to see
-your current agreements on file or to sign a new one.
+*  [Providing Feedback](#providing-feedback)
+    *  [Feature Requests](#feature-requests)
+    *  [Bug Reports](#bug-reports)
+*  [Contributing](#contributing)
+    *  [Contributor License Agreement (CLA)](#contributor-license-agreement-cla)
+    *  [How Libraries are Generated](#how-libraries-are-generated)
+    *  [Pull Request Submission Best Practices](#pull-request-submission-best-practices)
+    *  [Style Guidelines & Naming Conventions](#style-guidelines--naming-conventions)
+    *  [Claiming Requests](#claiming-requests)
+    *  [Testing](#testing)
+ * [Community Guidelines](#community-guidelines)
 
-You generally only need to submit a CLA once, so if you've already submitted one
-(even if it was for a different project), you probably don't need to do it
-again.
+## Providing Feedback
 
-## Code reviews
+This repository is specifically for the Google Ads API Client Library for Java. If you have feedback
+related to the Google Ads API, please submit it by clicking the "Send feedback" button in the top right
+corner of [this page](https://developers.google.com/google-ads/api/support).
 
-All submissions, including submissions by project members, require review. We
-use GitHub pull requests for this purpose. Consult
-[GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
-information on using pull requests.
+### Feature Requests
+
```

Comment:
Would be good to start FR + Bugs sections with some positive encouragement - currently the first thing in these sections is a prescriptive list of steps to take *before* submitting a ticket. We love getting feedback, and if folks read nothing else, I would like it to be telling them to go ahead and report the issue, even if that means we get more issues in the inbox. I'm happy to triage these requests as they come in and mark dupes etc.

Reply:
The opening paragraph contains that language. Do you think we need it here as well?